src/main/java/uk/nhs/digital/nhsconnect/nhais/model/mesh/WorkflowId.java

```diff
@@ -16,4 +19,11 @@
     public String toString() {
         return workflowId;
     }
+
+    public static WorkflowId fromString(String workflowId){
+        return Arrays.stream(WorkflowId.values())
+                .filter(workflow -> workflow.workflowId.equalsIgnoreCase(workflowId))
```

Comment:
Why not use exact matching and `Enum.valueof()` ?

Reply:
`WorkflowId` enum has "human friendly" values, like `REGISTRATION`, but the actual value that I map here is `NHAIS_REG`